doc/source/api/api.rst

```diff
@@ -0,0 +1,15 @@
+API
+---
+
+Overview
+========
+
+.. autosummary::
+    :nosignatures:
+    :toctree: api reference
```

Comment:
You don't want to have a space in this file path here. I would put it under `generated/`
```suggestion
    :toctree: generated/
```

Additionally, is the use of subdirectories really needed with the way the docs are organized now? It might be better to move everything into a flat organization, like:
```
doc/source
    api.rst
    credits.rst
    guidelines.rst
    index.rst
    install.rst
```
You can still make an `examples` subdirectory to store multiple files.

Last, none of the auto-generated files should be checked into the source tree, since they should be created by sphinx each time. If you do as I suggest above, remove them, and add `doc/source/generated` to `.gitignore`.